### PR TITLE
[CDAP-16076] removing checkboxes for provided arguments

### PR DESCRIPTION
**Pull request by itsanudeep**
JIRA: https://issues.cask.co/browse/CDAP-16076

Removing checkboxes for provided arguments. Provided arguments should have their value left empty. Adding a label that says if the arguments are left empty, they need to come from the pipeline. Any runtime arguments from preferences will be auto-populated.

### CODE REVIEW

File: cdap-ui/app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsButtons/PipelineRunButton.js [hunk 1]
```diff
@@ -48,18 +48,17 @@ export default class PipelineRunButton extends Component {
...
-      if (isMissingKeyValues) {
-        this.toggleRunConfigOption();
-      } else {
-        runPipeline();
-      }
+      let { runtimeArgs } = this.props;
+      // Arguments with empty values are assumed to be provided from the pipeline
+      runtimeArgs.pairs = runtimeArgs.pairs.filter((runtimeArg) => !runtimeArg.value);
+      let runtimeArgsMap = convertKeyValuePairsToMap(runtimeArgs.pairs);
+      runPipeline(runtimeArgsMap);
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsButtons/PipelineRunButton.js]: don't we still want the runtime argument modal to actually open? I think we should still open the modal but not force the user to fill everything out.

File: cdap-ui/app/cdap/components/PipelineDetails/PipelineRuntimeArgsDropdownBtn/RuntimeArgsModeless/index.js [hunk 2]
```diff
@@ -118,13 +106,16 @@ class RuntimeArgsModeless extends PureComponent {
...
-          disabled={this.state.savingAndRunBtnDisabled || this.state.saving}
+          disabled={this.state.saving}
           label="Run"
         />
       );
     };
     return (
       <div className="runtime-args-modeless">
+        <div className="arguments-label">
+          Specify runtime arguments, empty values need to come from the pipeline.
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/PipelineDetails/PipelineRuntimeArgsDropdownBtn/RuntimeArgsModeless/index.js]: i18n

File: cdap-ui/app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsButtons/PipelineRunButton.js (hunk 1 above)
Comment (itsanudeep) [reply cdap-ui/app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsButtons/PipelineRunButton.js]: So, we'd open the modal when user clicks `run` with empty arguments and they'd have to click `run` again to actually run the pipeline? Isn't that a bit confusing? I'll check with Lea and make these changes

Comment (itsanudeep) [reply cdap-ui/app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsButtons/PipelineRunButton.js]: Talked to Lea about this, leaving as is, when there are some runtime arguments are missing, we will show the popover. Will execute the pipeline if none are missing.

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 3]
```diff
@@ -2027,6 +2026,10 @@ features:
           clearAll: Clear All
           provided: Provided
           selectAll: Select All
+        RuntimeArgsModeless:
+          specifyArgs: Specify runtime arguments, empty values need to come from the pipeline.
```
Comment (yukiej) [inline cdap-ui/app/cdap/text/text-en.yaml]: nit: these should be in alphabetical order

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Comment by itsanudeep** [comment]
Build and test: https://builds.cask.co/browse/CDAP-UDUT467
RAT and Checkstyle: https://builds.cask.co/browse/CDAP-DRC9680

**Pull request merged**